etc/parameters/default.json

```diff
@@ -889,7 +889,8 @@
       "field_border_weight": 0.15,
       "line_walking_speed": 0.25,
       "arc_walking_speed": 0.2,
-      "rotation_penalty_factor": 0.4
+      "rotation_penalty_factor": 0.4,
+      "half_turning_time": 5.0
```

Comment:
This would require checking whether these have a linear correlation which I'm not 100% sure is the case.
Optimally this would be solved with #433. I'm not sure what is best because scaling it linearly seems a little like a hack and not doing so requires manually tuning this every time ```turn``` is adjusted.

Reply:
What benefit does the step planner give? I would agree to Rebaum that we can get this "fixed" in two steps: first this PR which does some approximation that is good enough, and then second #433. What do you think?

Reply:
I would prefer to merge something feature complete. Since it makes sense to test the turn rate anyway, it's an easy task to test it for two different settings.
Adjusting the values in the `step_planner` is a common task for tuning walking, so an "automatic" adjustment of the `half_turning_time` would be nice, since one probably wouldn't think of updating this value here. 

Expect you would add a comment in the default.json that the `half_turning_time` needs to be adjusted as well when adjusting the `turn` length.

Reply:
Json does not support comments

Reply:
You can in fact, if you use a variant - JSON5 for example. As I remember there is a serde crate for this too 😅.

I think HULKs C++ framework also had this capability.

Alternatively we could actually look at YAML or similar. But that's a different discussion IMO

Reply:
Ok, I will take some measurements and update the implementation.

Reply:
Did some measurements (numbers after the colon are the durations for one full rotation in seconds):

- turn=0.7: 8.8, 8.9, 9.3, 9.3, 9.7, 9.5, 9.2, 8.6, 10.0, 8.5
- turn=1.0: 5.8, 7.4, 6.3, 6.7, 5.6, 7.7
- turn=1.3: 5.4, 5.3, 5.0, 5.4, 5.7, 5.4, 5.38, 5.4, 5.8, 6.7

My conclusion: Very unreliable and not linear. Do we still want to do this?

Reply:
Hm okay, then let's do a constant value. Maybe something around `6.0`?

Reply:
Is now set to 6 seconds for a full rotation (3 seconds for a half rotation)